src/status_im/signals/core.cljs

```diff
@@ -97,7 +98,12 @@
            (when (and completed-pairing? sender?)
              {:dispatch [:syncing/clear-states]})
            (when (and completed-pairing? receiver?)
-             {:dispatch [:multiaccounts.login/local-paired-user]}))))
+             {:dispatch [:multiaccounts.login/local-paired-user]})
```

Comment:
this looks more like `cond` no?

Reply:
there should be no merge, but something like {:dispatch (cond ....)}

Reply:
@flexsurfer - Yes, This is more like `cond`itional dispatches. 
We can refactor this to:

```clojure
(merge {:db {}}
       (cond 
             (and completed-pairing? sender?) ;; case 1
             {:dispatch [:something]}

             (and completed-pairing? receiver?)  ;; case 2
             {:dispatch [:something-else]}))
```

> there should be no merge, but something like {:dispatch (cond ....)}

In some cases, there are no dispatches, which will produce `nil`, producing a warning.

 I will do the refactor in a follow-up PR.